src/main/java/me/zeroeightsix/kami/module/modules/player/Fastuse.kt

```diff
@@ -24,6 +24,7 @@ class Fastuse : Module() {
     private val delay = register(Settings.integerBuilder("Delay").withMinimum(0).withMaximum(20).withValue(0).build())
     private val all = register(Settings.b("All", false))
     private val bow = register(Settings.booleanBuilder().withName("Bow").withValue(true).withVisibility { !all.value }.build())
+    private val chargeState = register(Settings.integerBuilder("Fire bow on charge state").withMinimum(0).withMaximum(20).withValue(3).build())
```

Comment:
could you rename this to like "Bow Charge" and make it only visible when bow or all are enabled?